models/user_model.py

```diff
@@ -93,6 +94,10 @@ def delete_user(username):
         db.session.commit()
         return done
 
+    def update_last_login(self):
```

Comment:
update_last_login function needs unit test coverage and error handling for database operations